src/rust/grapl-web-ui/tests/api/auth/check_login.rs

```diff
@@ -0,0 +1,27 @@
+#[actix_web::test]
+async fn auth_unauthenticated_check_login() -> eyre::Result<()> {
+    let app = crate::test_app::TestApp::init().await?;
+
+    let response = app.post("api/auth/checkLogin").send().await?;
+
+    println!("{:?}", response);
```

Comment:
don't love the bare println but whatever

Reply:
Yeah, they could at least say what they are. 👍